plugins/azure/virtualnetworks/noVPNgateways.js

```diff
@@ -0,0 +1,67 @@
+var async = require('async');
+var helpers = require('../../../helpers/azure');
+
+module.exports = {
+    title: 'No VPN Gateways',
+    category: 'Virtual Networks',
+    domain: 'Network Access Control',
+    description: 'Ensures that virtual network gateways type is not VPC.',
+    more_info: 'Use ExpressRoute as gateway type instead of VPN for virtual network gateways to meet your organization\'s security and compliance requirements.',
```

Comment:
```suggestion
    more_info: 'Use ExpressRoute as gateway type instead of VPN for virtual network gateways to meet your organization\'s security and compliance requirements.',
```
elaborate a little